pom.xml

```diff
@@ -118,6 +120,32 @@
 					<artifactId>maven-failsafe-plugin</artifactId>
 					<version>${maven-surefire-plugin-version}</version>
 				</plugin>
+				<!-- 
+					Spotless info:
+					  - How to preview what `mvn spotless:apply` will do: https://github.com/diffplug/spotless/tree/main/plugin-maven#how-do-i-preview-what-mvn-spotlessapply-will-do
+					  - Apply Spotless to specific files: https://github.com/diffplug/spotless/tree/main/plugin-maven#can-i-apply-spotless-to-specific-files
+				-->
+				<plugin>
+					<groupId>com.diffplug.spotless</groupId>
```

Comment:
Indentation in this file: Please stay consistent, and don't use tabs. 4 spaces??

Reply:
Done. Fixed the inconsistent indentation.

That being said... since the rest of the file is using tabs, I made one commit to make the indentation consistent with the tab-based indents.  The subsequent commit converts the entire file from tabs to spaces.

It seems that multiple *.xml files in the repo use tabs, not spaces.  Interested in adding a [4-space indent rule for certain files](https://github.com/diffplug/spotless/tree/main/plugin-maven#generic-steps) to the formatter / style checker?

Reply:
> It seems that multiple *.xml files in the repo use tabs, not spaces. Interested in adding a [4-space indent rule for certain files](https://github.com/diffplug/spotless/tree/main/plugin-maven#generic-steps) to the formatter / style checker?

Yes, I think so.

@josh-hadley it looks like the tab checker does not look at XML files?

We probably can't enforce much about .txt files. Or maybe none of them even use indentation.

Reply:
Right, tab checker only looks at files ending with `.java`. Easy enough to add XML or others if desired.

Reply:
our xml files [pom.xml, build.xml, all cldr data files] have traditionally used tabs.  Is there a reason to change?

Reply:
Looks like icu4j's build.xml uses spaces, though

Reply:
Tabs are generally trouble because there is not a standard advance width for them. Many tools default to tab stops every 8 columns, but some people set some of their tools to 4, and when files use a mix of tabs and spaces (which often happens) the indentation is all messed up in one editor / code review tool or another. Or indentation with tabs and vertically aligned comments at the ends of lines. That's why we try to avoid tabs.